tests/import_schemas.py

```diff
@@ -0,0 +1,183 @@
+"""
+A script to generate the schemas for the GA4GH protocol
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import glob
+import shutil
+import os.path
+import tarfile
+import tempfile
+import re
+
+import avro.schema
+
+import utils
+
+
+class SchemaClass(object):
+    """
+    Class to convert an Avro JSON definition of a GA4GH type into the
+    corresponding Python class.
+    """
+    def __init__(self, sourceFile):
+        self.sourceFile = sourceFile
+        with open(sourceFile) as sf:
+            self.schemaSource = sf.read()
+            self.schema = avro.schema.parse(self.schemaSource)
+        self.name = self.schema.name
+
+    def getFields(self):
+        """
+        Returns the list of avro fields sorted in order of name.
+        """
+        return sorted(self.schema.fields, key=lambda f: f.name)
+
+    def isSearchRequest(self):
+        """
+        Returns True if the class we are converting is a subclass of
+        SearchRequest, and False otherwise.
+        """
+        return re.search('Search.+Request', self.name) is not None
+
+    def isSearchResponse(self):
+        """
+        Returns True if the class we are converting is a subclass of
+        SearchResponse, and False otherwise.
+        """
+        return re.search('Search.+Response', self.name) is not None
+
+    def checkEmbeddedTypes(self):
+        """
+        Returns the set of embedded types in this class.
```

Comment:
Comments don't agree with the name here. Do we _use_ the returned value ever? If not, why build the list? People who have never seen our server repo are going to come in and try to understand this script. It's going to look very confusing and weird if we don't make it more focused on what it's supposed to be doing: *checking* the schemas, not generating Python code.